docs/events.md

```diff
@@ -51,10 +51,15 @@ retrieving those events using the `kubectl describe` command. Tekton can also em
 When you [configure a sink](install.md#configuring-cloudevents-notifications), Tekton emits
 events as described in the table below.
 
-Resource  |Event    |Event Type
-:---------|:-------:|:----------------------------------------------------------
-`TaskRun` | `Started` | `dev.tekton.event.taskrun.started.v1`
-`TaskRun` | `Running` | `dev.tekton.event.taskrun.runnning.v1`
-`TaskRun` | `Condition Change while Running` | `dev.tekton.event.taskrun.unknown.v1`
-`TaskRun` | `Succeed` | `dev.tekton.event.taskrun.successful.v1`
-`TaskRun` | `Failed` | `dev.tekton.event.taskrun.failed.v1`
+Resource      |Event    |Event Type
+:-------------|:-------:|:----------------------------------------------------------
+`TaskRun`     | `Started` | `dev.tekton.event.taskrun.started.v1`
+`TaskRun`     | `Running` | `dev.tekton.event.taskrun.runnning.v1`
+`TaskRun`     | `Condition Change while Running` | `dev.tekton.event.taskrun.unknown.v1`
+`TaskRun`     | `Succeed` | `dev.tekton.event.taskrun.successful.v1`
+`TaskRun`     | `Failed`  | `dev.tekton.event.taskrun.failed.v1`
+`PipelineRun` | `Started` | `dev.tekton.event.pipelinerun.started.v1`
```

Comment:
I think this is left over from another PR.

Reply:
This PR depends on https://github.com/tektoncd/pipeline/pull/2938 and since github does not provide any mechanism to declare dependencies between PRs, I had to rebase this one on top of the other (there are two commits in the PR).
Once #2938 is merge I will drop the first commit.

Reply:
Rebased